src/scenic/syntax/compiler.py

```diff
@@ -253,6 +256,16 @@ def visit_Require(self, node: s.Require):
             )
         )
 
+    def visit_Abort(self, node: s.Abort):
+        return ast.copy_location(
+            ast.Return(
+                ast.Attribute(
```

Comment:
Did you mean to use `abortFlag` here?

Reply:
You're right. Thanks for the catch. Fixed in d47c02c98b8d1f298906154cf7640089385d9164